client/src/main/java/timely/model/ObjectSizeOf.java

```diff
@@ -87,15 +86,8 @@ public static long getObjectSize(Object o, Set<ObjectInstance> visited, Stack<Ob
                     visited.add(oi);
                     try {
                         if (useSizeInBytesMethod) {
-                            try {
-                                if (o instanceof ObjectSizeOf) {
-                                    size = ((ObjectSizeOf) o).sizeInBytes();
-                                } else {
-                                    Method sizeInBytes = o.getClass().getMethod("sizeInBytes", (Class<?>[]) null);
-                                    size = (Long) sizeInBytes.invoke(o);
-                                }
-                            } catch (Throwable t) {
-                                // ok, lets do this the hard way...
+                            if (o instanceof ObjectSizeOf) {
```

Comment:
It'd be cool to create a wrapper for HashMap<Key,Value> which implements ObjectSizeOf. I know we might not do it soon but is it worth opening an issue for that?

Reply:
It seems that could be worthwhile for a separate ticket.  I wanted to get a low effort performance improvement with this MR.